integreat_cms/cms/templates/content_revisions.html

```diff
@@ -6,24 +6,22 @@
     {% load rules %}
     <div class="mb-6">
         <h1 class="heading">
-            {% with page_translations.first.title as page_title %}
-                {% blocktrans %}Page versions of "{{ page_title }}"{% endblocktrans %}
+            {% with content_translations.first.title as content_title %}
+                {% blocktrans %}"Versions of {{ content_title }}"{% endblocktrans %}
             {% endwith %}
         </h1>
-        <a href="{% url 'edit_page' page_id=page.id region_slug=request.region.slug language_slug=language.slug %}"
-           class="btn btn-outline">
-            <i icon-name="arrow-left-circle" class="align-top"></i> {% translate "Back to the page form" %}
+        <a href={{ form }} class="btn btn-outline">
```

Comment:
Quotes are missing and `{{ form }}` usually holds an actual form object and not a URL, so I would rename this variable.